src/services/pcn-bridge/src/Fdb.cpp

```diff
@@ -0,0 +1,63 @@
+/**
+ * bridge API generated from bridge.yang
+ *
+ * NOTE: This file is auto generated by polycube-codegen
+ * https://github.com/polycube-network/polycube-codegen
+ */
+
+// TODO: Modify these methods with your own implementation
+
+#include "Fdb.h"
+#include "Bridge.h"
+
+Fdb::Fdb(Bridge &parent, const FdbJsonObject &conf) : FdbBase(parent) {}
```

Comment:
Just asking: is this really the code generated or maybe you remove a piece when organizing the commits?

Reply:
Yes, this is the code generated yesterday from the latest datamodel. The only thing i did was running the check-style script!

Reply:
Ok, there is then some bug there because the initialization code should be there.